cmd/promtool/main.go

```diff
@@ -200,6 +200,12 @@ func main() {
 	dumpMaxTime := tsdbDumpCmd.Flag("max-time", "Maximum timestamp to dump.").Default(strconv.FormatInt(math.MaxInt64, 10)).Int64()
 	dumpMatch := tsdbDumpCmd.Flag("match", "Series selector.").Default("{__name__=~'(?s:.*)'}").String()
 
+	tsdbRelabelCmd := tsdbCmd.Command("relabel", "Relabel TSDB block.")
+	relabelPath := tsdbRelabelCmd.Arg("db path", "Database path (default is "+defaultDBPath+").").Default(defaultDBPath).String()
+	relabelBlockID := tsdbRelabelCmd.Arg("block id", "BlockID to relabel.").String()
+	relabelConfig := tsdbRelabelCmd.Arg("file", "Relabel config file to apply.").String()
```

Comment:
As such, having a default for the "db path" argument doesn't make sense because there are multiple mandatory arguments following.

I would suggest allowing for an empty block id flag (which would be a list of string). If the list of blocks is empty, we can loop over all blocks. 

Additionally, I would recommend making the relabel config file a flag that defaults to relabel.yml.

Reply:
I see you have a solution for this.